app/src/main/java/com/nextcloud/talk/activities/CallActivity.kt

```diff
@@ -663,7 +675,38 @@ class CallActivity : CallBaseActivity() {
                 ).show()
             }
         }
-        binding!!.hangupButton.setOnClickListener { hangup(true) }
+
+        if (isGroupConversation) {
+            binding!!.hangupButton.setOnClickListener {
+                hangup(true, null)
+            }
+        }
+
+        if (isOneToOneConversation && isModerator) {
+            binding!!.hangupButton.setOnClickListener {
+                showPopupMenu()
```

Comment:
> One-to-one:
> Short/Normal press: "End call" (Label (if shown) is short, but behaviour is "End call for everyone")

so the action here must be
`hangup(true, true)`

Reply:
Short/Normal press of hangupButton in one-one conversation should show a button ("End call for everyone"). If you click this button, then only end call for everyone. This is how ioS does it.